Updating Azure Kinect SDK to 1.4.1 & Github templates (#13)
secretlocation/azure-kinect-unreal: README.md

## SDKs

- [Azure Kinect SDK 1.4.1](https://docs.microsoft.com/en-us/azure/kinect-dk/sensor-sdk-download)
- [Azure Kinect Body Tracking SDK 1.0.1](https://docs.microsoft.com/en-us/azure/kinect-dk/body-sdk-download)

## To get up and running

1. Add the `<Azure Kinect Body Tracking SDK installation>/tools` folder to the `Path` variable in `Environment Variables` for User or System.
2. If you want to enable Azure Kinect logging, run the following 4 commands in Command Prompt as an admin:

>    `setx K4A_ENABLE_LOG_TO_A_FILE k4a.log /M`
>    `setx K4A_LOG_LEVEL w /M`
>    `setx K4ABT_ENABLE_LOG_TO_A_FILE k4abt.log /M`
>    `setx K4ABT_LOG_LEVEL w /M`

Reviewer: ```suggestion
setx K4A_ENABLE_LOG_TO_A_FILE k4a.log /M
setx K4A_LOG_LEVEL w /M
setx K4ABT_ENABLE_LOG_TO_A_FILE k4abt.log /M
setx K4ABT_LOG_LEVEL w /M
```

Formatting suggestion since they are shown on one single line.

Author: I'll fix the single line - should be multiline

## Notes

The Azure Kinect project will need to reference the following dll to function:
```
k4a.dll  
k4abt.dll  
depthengine_2_0.dll  
dnn_model_2_0.onnx  
onnxruntime.dll  
cudnn64_7.dll  
cudart64_100.dll  
cublas64_100.dll  
```

We've included some of these dlls in `Project/Plugins/AzureKinectUnreal/Binaries/Win64`, just to allow the project to compile when opened directly after cloning:
```
k4a.dll
k4abt.dll
depthengine_2_0.dll
dnn_model_2_0.onnx
onnxruntime.dll  
```

If you're getting errors with dlls or libraries not loading properly, check if the libraries that exist in these places are all of the same version.  

> Keep in mind that you only need the dlls/libraries in one of these places:

Reviewer: ```suggestion
Kindly note that the `dlls/libraries` should **only be in one** of the below locations.
```

>  - `<Azure Kinect Body Tracking SDK installation>/tools`, if this has been added to your PATH in the environmental variables. This is the best place to keep them, as it's the location all Azure Kinect stuff uses. You can have the libraries exist only here and nowhere else, and it should work just fine.
>  - `<Unreal Installation Folder>/Engine/Binaries/Win64/`. You don't have to put any dlls or libraries here if you have the body tracking sdk tools in your PATH
>  - `Plugins/AzureKinectUnreal/Binaries/Win64/`. We've included some dlls here to make sure it compiles from a straight clone. You can delete them if they mismatch with your body tracking SDK version and it's added to your PATH.

Reviewer: ```suggestion
- `<Azure Kinect Body Tracking SDK installation>/tools`, if this has been added to your PATH in the environmental variables. This is the best place to keep them, as it's the location all Azure Kinect stuff uses. You can have the libraries exist only here and nowhere else, and it should work just fine. 
**(or)**
 - `<Unreal Installation Folder>/Engine/Binaries/Win64/`. You don't have to put any dlls or libraries here if you have the body tracking sdk tools in your PATH. 
 **(or)**
 - `Plugins/AzureKinectUnreal/Binaries/Win64/`. We've included some dlls here to make sure it compiles from a straight clone. You can delete them if they mismatch with your body tracking SDK version and it's added to your PATH.
```

Suggestion to add **(or)** to be clear.

Kindly check the `Output Log` window and if the issue is something related to the `Device` then check the `k4a.log` and if it is related to the `Body Tracking` then check the `k4abt.log`. These log files can be found in `<unreal_installation_folder>/Engine/Binaries/Win64/`.

If these log files are not present then kindly check [here](https://docs.microsoft.com/en-us/azure/kinect-dk/troubleshooting#collecting-logs) in order to enable the logs.

#### The project can't find the Azure Kinect

> Check if the Azure Kinect shows up under the *Cameras* section of the Device Manager. It sometimes doesn't connect on startup, which can be fixed by unplugging the USB core and plugging it back in. The white LED on the front will show it connecting.

Reviewer: ```suggestion
Kindly check if the Azure Kinect shows up under the *Cameras* section of the Device Manager. It sometimes doesn't connect on startup, which can be fixed by unplugging the USB core and plugging it back in. The white LED on the front will show it connecting.
```
